README.md

```diff
@@ -1 +1,41 @@
-# Capstone-Attempt-1
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# DL Global Summit
+
+> In this project I have recreated one of Cindy Shin's (link in the acknowledgments section) pages but with my own content. The two pages recreated are the home page and the about page. The recreation is made to be similar to the original design but not exact.
+
+
+## Built With
+
+- HTML, SCSS, JavaScript
+
+## Live Demo 
+
+[Live Demo Link](https://dodoburner.github.io/DL-Global-Summit/)
+
```

Comment:
- [ ] Please it will be nice to add explicit steps on how  to clone and run you project locally, as this will be of help to non technical users and some of them are potential recruiters.